src/test/java/com/amazon/opendistroforelasticsearch/security/auditlog/compliance/ComplianceConfigTest.java

```diff
@@ -35,7 +35,6 @@ public void testDefault() {
         assertFalse(complianceConfig.shouldLogReadMetadataOnly());
         assertFalse(complianceConfig.shouldLogWriteMetadataOnly());
         assertFalse(complianceConfig.shouldLogDiffsForWrite());
-        assertSame(WildcardMatcher.NONE, complianceConfig.getImmutableIndicesMatcher());
```

Comment:
Can deleted test be moved to `OpenDistroSecurityFilterTest` (create if if necessary)?

Reply:
I have migrated all the tests